herciuana_Task2/DynamicStringArray.cpp

```diff
@@ -0,0 +1,110 @@
+#include "DynamicStringArray.hpp"
+
+DynamicStringArray::DynamicStringArray() : dynamicArray(nullptr), size(0)
+{
+}
+
+int DynamicStringArray::get_Size() const
+{
+    return size;
+}
+
+void DynamicStringArray::addEntry(const string &String)
+{
+    string *Array = new string[size + 1];
+    for (int i = 0; i < size; i++)
+    {
+        Array[i] = dynamicArray[i];
+    }
+    Array[size] = String;
+    size++;
+    delete[] dynamicArray;
+    dynamicArray = Array;
+}
+
+bool DynamicStringArray::deleteEntry(const string &String)
+{
+    bool r = false;
+    int i;
+    for (int i = 0; i < size; i++)
+    {
+        if (dynamicArray[i] == String)
+        {
+            r = true;
+        }
+    }
+    if (r == true)
```

Comment:
Is not easier to return r?
Also this method is incomplete, doesn't delete the entry(string) in the case that is found.